app/components/QuestionRow/QuestionRow.jsx

```diff
@@ -14,6 +14,8 @@ import { getDateData } from 'app/utils/dates/timeOperations';
 import ACTIONS from 'app/utils/actions';
 import { CircleIcon, DateContainer } from 'app/components/QuestionResponderInfo/QuestionResponderInfo.Styled';
 import Switch from 'app/components/Switch';
+import { MdLockOpen } from 'react-icons/md';
+import Button from '../Atoms/Button';
```

Comment:
Would it be possible to modify this in order to adhere to the established standard?

Reply:
@GeorginaFraire Sure, weirdly the eslint rules do not warned about this but I will do

Reply:
Done @GeorginaFraire 😄